src/api/workspaces/index.ts

```diff
@@ -49,6 +50,15 @@ export async function leaveWorkspace(workspaceId: string): Promise<boolean> {
   return (await api.callOld(MUTATION_LEAVE_WORKSPACE, { workspaceId })).leaveWorkspace;
 }
 
+/**
+ * Leave workspace
+ *
+ * @param {string} workspaceId - id of workspace to delete
+ */
+export async function deleteWorkspace(workspaceId: string): Promise<boolean> {
```

Comment:
Incorrect return value. The method should return an object of type `Promise<APIResponse<{ anydata }>>`.

The calling of `(await api.call()).data.deleteWorkspace` is unsafe call. There will be an error if a server returns an error (for example, network error). 

See `createWorkspace` for example